Examples/Python/RunUseCase.py

```diff
@@ -42,7 +42,7 @@
 parser.add_argument("--start_with_image_and_segmentation_data", help = "use images and segmentations data for preprocessing", action="store_true")
 parser.add_argument("--use_single_scale", help="Single scale or multi scale optimization", action="store_true")
 parser.add_argument("--tiny_test", help="Run as a short test", action="store_true")
-parser.add_argument("shapeworks_path", help="Path to ShapeWorks executables (default: "+binpath+")", nargs='?', type=str, default=binpath)
+parser.add_argument("--shapeworks_path", help="Path to ShapeWorks executables (default: "+binpath+")", nargs='?', type=str, default=binpath)
```

Comment:
This change breaks the PythonTests that assumes the last optional argument is the shapeworks path.  This argument did not previous need "--shapeworks_path"